aws-lc-fips-sys/builder/sys_bindgen.rs

```diff
@@ -56,7 +56,8 @@ fn prepare_bindings_builder(manifest_dir: &Path, options: &BindingOptions) -> bi
                 .join("rust_wrapper.h")
                 .display()
                 .to_string(),
-        );
+        )
+        .parse_callbacks(Box::new(bindgen::CargoCallbacks::new()));
```

Comment:
These [callbacks](https://docs.rs/bindgen/latest/bindgen/struct.CargoCallbacks.html) would only be needed if someone were modifying the AWS-LC headers in the submodule under "aws-lc-sys", which should never happen. But I don't see much harm other than a few dozen more "rerun-if-changed" lines going to the build script output.